baremetal_network_provisioning/ml2/hp_network_provisioning_driver.py

```diff
@@ -161,20 +137,51 @@ def update_port(self, port_dict):
         SDN controller for provision VLAN on switch port where bare metal
         is connected.
         """
-        return
+        LOG.debug("update_port with port dict %(port_dict)s",
+                  {'port_dict': port_dict})
+        port_id = port_dict['port']['id']
+        bind_requested = port_dict['port']['bind_requested']
+        update_dict = {'neutron_port_id': port_id,
+                       'bind_requested': bind_requested}
+        db.update_hp_ironic_swport_map_with_bind_req(self.context,
+                                                     update_dict)
 
-    def delete_port(self, port_id):
+    def delete_port(self, port_dict):
```

Comment:
even though I am parsing only  the neutron_port_id from the delete_port from back-end driver, I thought it will be place holder for other back-end drivers.